github/event_types.go

```diff
@@ -486,7 +486,7 @@ type PullRequestEvent struct {
 	// the pull request was closed with unmerged commits. If the action is "closed"
 	// and the merged key is true, the pull request was merged.
 	Action      *string      `json:"action,omitempty"`
-	Number      *int         `json:"number,omitempty"`
+	Number      *int64       `json:"number,omitempty"`
```

Comment:
Why is this `Number` field changed to `int64`? It's not an ID. I'm guessing this was an accident, right?

Reply:
My bad; this is a mistake on my part. The `Number` field should not be changed.
Let me rectify it.